_posts/2016-05-24-SpringIO16-Spring-Rest-Docs.md

```diff
@@ -0,0 +1,339 @@
+---
+layout: post
+authors: [kevin_van_houtte]
+title: 'Spring I/O 16: Test-driven documentation with Spring REST Docs'
+image: /img/springio.jpg
+tags: [Spring IO,Spring,Conference]
+category: conference
+comments: true
+---
+
+## Spring IO 2016
+The main focus this year was definitely about Cloud, Reactive and MicroServices.
+But to complete this story we will need to document the REST API. 
+Keep calm, you don't have to do it manually! 
+Spring made it easy for us with Spring Rest Docs! 
+This year in Spring IO, Andy Wilkinson himself talked about why, how and when Spring Rest Docs are being used. 
+Last but not least, he talked about the new features that came out in version 1.1.
+Since I implemented Spring REST Docs in a project, I'll use examples from my experiences.
+
+
+<img class="p-image float-image" width="200" alt="Andy Wilkinson" src="/img/rest-docs/andywilkinson.jpg">
+
+### Andy Wilkinson
+
+Andy is a Spring Boot, REST docs committer and Spring IO platform lead at Pivotal. 
+You can find him on Twitter using the handle [@ankinson](https://twitter.com/ankinson).
+
+<blockquote class="clear"><p>
+Writing documentation is critical in the world of development. 
+It is used to make a accurate and straight declaration and intent of what the service has to offer. 
+Front-End developers will be able to know which endpoints they have to call and receive the right resource. 
+Now we all know it's tedious for developers to write documentation...
+Price your self lucky! Spring Rest Docs will make your life easier.
```

Comment:
Nederengels :-)
Would replace it with something else, maybe:
"It's your lucky day!"